test/Datadog.Trace.Tests/Agent/Jaeger/JaegerExporterTests.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using Datadog.Trace.Agent.Jaeger;
+using Xunit;
+
+namespace Datadog.Trace.Tests.Agent.Jaeger
+{
+    public class JaegerExporterTests
```

Comment:
for me these tests are useless. They test only internal code and I do have any feedback if the public `SendTraces` method works. It only makes refactoring harder.

Reply:
Supporting to leave it as is. Better to have original than none.
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/78#discussion_r590901164